scipy/stats/_continuous_distns.py

```diff
@@ -207,6 +208,8 @@ class alpha_gen(rv_continuous):
     %(example)s
 
     """
+    _support_mask = rv_continuous._open_support_mask
```

Comment:
The open_support_mask is used in the cdf, logcdf, sf and logsf methods in order to not change the behaviour from the previous version. The reasons I thought it was a good idea was:
 1. Reuse of code in cdf, logcdf, sf and logsf and as well as all the distributions that have open_support.
 2. _open_support(x) is more readable than  self.a < x & self.b < b in cdf, logcdf, sf and logsf.
 3.  Code will be shorter due to less  copy/paste code.

If you still think it is better to copy paste code to have like it was in cdf, logcdf, sf and logsf 
and explicitly paste in the  _support_mask for every distribution it is needed I will do so.